flujo/validation/linters.py

```
@@ -1759,6 +1759,399 @@ def analyze(self, pipeline: Any) -> Iterable[ValidationFinding]:
+
+        # Import step types lazily to avoid circular dependencies
+        try:
+            from ..domain.dsl.step import HumanInTheLoopStep as _HITLStep
+        except Exception:
+            _HITLStep = None  # type: ignore
+
+        if _HITLStep is None:
+            return out
+
+        def _check_for_hitl_in_steps(
+            steps: list[Any], context_chain: list[str], parent_step_name: str | None = None
```

Ruff (0.13.3)

Unused function argument: `parent_step_name`

(ARG001)